feature/nuc-226-mkapi-transfer-slider-component-to-vuexp

mkapi opened the pull request:
NUC-226 Implementation

osmanertem commented on docs/components/slider.md:
```diff
@@ -0,0 +1,41 @@
+| ----------- | --- | ------ |
+| valueChange | ✔   | ✔      |
+
+## References
```
references section should be removed

mkapi replied:
removed references section in d3c44a61e4d5a2c80e231093710514d2ba83e79a

osmanertem commented on docs/components/slider.md:
```diff
@@ -0,0 +1,41 @@
+
+| Name     | Type   | Default | Web | Mobile |
+| -------- | ------ | ------- | --- | ------ |
+| value    | Number | `0`     | ✔   | ✔      |
```
description column should be added on both props and events section

mkapi replied:
Added description in d3c44a61e4d5a2c80e231093710514d2ba83e79a

osmanertem commented on docs/.vuepress/components/SliderDoc.vue:
```diff
@@ -0,0 +1,46 @@
+<template>
```
there should be a live demo part on document. 
It can be implemented in a separate vue file like 
SliderDocLiveDemo.vue

mkapi replied:
Updated live demo in d3c44a61e4d5a2c80e231093710514d2ba83e79a

mkapi commented:
@osmanertem Removed references section. Added description to props and events. Updated slider live demo. Please refer to d3c44a61e4d5a2c80e231093710514d2ba83e79a

Also screenshot of doc page:
![Screenshot_2019-05-02 Slider VueXP Documentation](https://user-images.githubusercontent.com/13813553/57070641-be23c480-6ce0-11e9-8067-efdf01d6d064.png)

ahmetceylan commented on docs/components/slider.md:
```diff
@@ -0,0 +1,47 @@
+---
+sidebarDepth: 0
+---
```
please use documentation template to be consistent with other pages

mkapi replied:
Updated doc page according to template with https://github.com/vuexp/vuexp/pull/14/commits/87816117337d13c9694b6716c379a40eb3f6f072

Updated doc page:
![image](https://user-images.githubusercontent.com/13813553/57077208-1fa15e80-6cf4-11e9-84be-bfedfa4df8cb.png)
